### PR TITLE
fix: Fixes for pre changelog generation

### PR DESCRIPTION
**Pull request by TriPSs**

### CODE REVIEW

**Comment by TriPSs** [comment]
@kurnev I created this PR, could you also check it out?

File: src/helpers/bumpVersion.js [hunk 1]
```diff
@@ -44,5 +46,22 @@ module.exports = (releaseType, version) => {
...
+  const preChangelogGenerationFile = core.getInput('pre-changelog-generation')
+
+  let newVersion = `${major}.${minor}.${patch}`
+
+  if (preChangelogGenerationFile) {
+    const preChangelogGenerationScript = requireScript(preChangelogGenerationFile)
+
+    // Double check if we want to update / do something with the version
+    if (preChangelogGenerationScript && preChangelogGenerationScript.preVersionGeneration) {
+      const modifiedVersion = preChangelogGenerationScript.preVersionGeneration(newVersion)
```
Comment (kurnev) [inline src/helpers/bumpVersion.js:58]: Should not we `await` the script in case of async function?

File: src/index.js [hunk 2]
```diff
@@ -102,24 +105,26 @@ async function run() {
...
-        const newVersionAndTag = await require(path.resolve(process.cwd(), preChangelogGeneration)).preChangelogGeneration({
-          tag: gitTag,
-          version: newVersion,
-        })
+      if (preChangelogGenerationFile) {
+        const preChangelogGenerationScript = requireScript(preChangelogGenerationFile)
+
+        // Double check if we want to update / do something with the tag
+        if (preChangelogGenerationScript && preChangelogGenerationScript.preTagGeneration) {
+          const modifiedTag = preChangelogGenerationScript.preTagGeneration(gitTag)
```
Comment (kurnev) [inline src/index.js:123]: `await` here as well?

**Review by kurnev — COMMENTED** [review]
Looks good to me!

**Pull request merged**